### PR TITLE
Altered .gitignore and removed .vscode/settings.json from repo

### PR DESCRIPTION
**Pull request by bryanjnelson**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,7 +26,7 @@ package-lock.json
 
 # IDE - VSCode
 .vscode/*
-!.vscode/settings.json
+.vscode/settings.json
```
Comment (CarsonF) [inline .gitignore:29]:
```suggestion
```

**Review by CarsonF — CHANGES_REQUESTED** [review]

**Review by CarsonF — COMMENTED** [review]
Do you want to ignore tasks or launch while we are here too or you want to share those?

**Review by bryanjnelson — COMMENTED** [review]
Nah, those can be left alone for now.

**Review by CarsonF — APPROVED** [review]
LGTM

**Pull request merged**